source/components/recipe-card-delete.js

```diff
@@ -6,18 +6,45 @@ class RecipeCardDelete extends HTMLElement {
   constructor() {
     super();
     this.attachShadow({ mode: "open" });
+  }
+
+  /**
+   * Sets the recipe object for a given card and sets up content
+   */
+  set recipe(recipeObj) {
+    if (!recipeObj) {
+      return;
+    }
 
     const stylesheet = document.createElement("link");
     stylesheet.rel = "stylesheet";
     stylesheet.href = "/source/styles/recipe-card-delete.css";
 
     const recipeCard = document
       .getElementById("recipe-card-delete-template")
-      .cloneNode(true);
+      .content.cloneNode(true);
+
+    // store object for access later
+    this.recipeObj = recipeObj;
+
+    // set up card content
+    let titleText = recipeCard.querySelector("h1");
+    titleText.textContent = recipeObj.title;
 
+    let descriptionText = recipeCard.querySelector(".description");
+    descriptionText.textContent = recipeObj.description;
```

Comment:
I now understand why the Spoonacular descriptions have weird tags. It's because Spoonacular is giving us the HTML, so change this to `.innerHTML` for proper formatting. I'll have to make an issue for Sprint 2 to fix this error in the other cards as well
```suggestion
    descriptionText.innerHTML = recipeObj.description;
```